ingest/rules/upload_from_fauna.smk

```diff
@@ -1,8 +1,9 @@
 rule download_segment:
     output:
-        sequences = "data/{segment}.fasta",
+        sequences = "data/fauna/{segment}.fasta",
     params:
         fasta_fields = "strain virus accession collection_date region country division location host domestic_status subtype originating_lab submitting_lab authors PMID gisaid_clade h5_clade",
+        output_dir = "data/fauna",
```

Comment:
_non-blocking_
The Snakemake `params` can access the output, so we can pull out the `output_dir` directly

```suggestion
        output_dir = lambda wildcards, output: Path(output.sequences).parent,
```

Reply:
Updated in https://github.com/nextstrain/avian-flu/pull/36/commits/5e903033875bfcca53a61f6a33436ef5d048ff67